arangod/RocksDBEngine/RocksDBSettingsManager.cpp

```diff
@@ -279,7 +277,6 @@ void RocksDBSettingsManager::loadSettings() {
 
 /// earliest safe sequence number to throw away from wal
 rocksdb::SequenceNumber RocksDBSettingsManager::earliestSeqNeeded() const {
-  READ_LOCKER(guard, _rwLock);
```

Comment:
I'm somewhat concerned about removing the lock. At the very least, I think then that `_lastSync` needs to be atomic.